File: 백준/Bronze/9498. 시험 성적/시험 성적.c
#include <stdio.h>

int main(void) {
    int score = 0;
    scanf("%d", &score);
    if (score >= 90)
        printf("A\n");
    else if (90 > score && score >= 80)
        printf("B\n");
    else if (80 > score && score >= 70)
        printf("C\n");
    else if (70 > score && score >= 60)
        printf("D\n");
    else
        printf("F\n");
    return 0;
}

File: 백준/Bronze/9498. 시험 성적/시험 성적.cs
using System;

class Program {
    static void Main(string[] args) {
        string str = Console.ReadLine();
        int score = int.Parse(str);
        switch(score/10) {
        case 10:
        case 9:
            Console.WriteLine("A");
            break;
        case 8:
            Console.WriteLine("B");
            break;
        case 7:
            Console.WriteLine("C");
            break;
        case 6:
            Console.WriteLine("D");
            break;
        default:
            Console.WriteLine("F");
            break;
        }
    }
}